src/BPEssentials/Commands/Moderation/Jail.cs

```diff
@@ -0,0 +1,39 @@
+﻿using BPEssentials.Abstractions;
+using BPEssentials.ExtensionMethods;
+using BrokeProtocol.Entities;
+using BrokeProtocol.Required;
+using BrokeProtocol.Utility;
+using BrokeProtocol.Utility.Jobs;
+using BrokeProtocol.Utility.Networking;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using UnityEngine;
+
+namespace BPEssentials.Commands
+{
+	public class Jail : Command
+	{
+		public void Invoke(ShPlayer player, ShPlayer target, float timeInSeconds)
+		{
+			ShJail jail = target.manager.jails.FirstOrDefault();
+			if (jail)
+			{
+				if (target.IsDead || target.job is Prisoner)
+				{
+					return;
+				}
+				Transform getPositionT = jail.GetPositionT;
+				target.svPlayer.SvTrySetJob(JobIndex.Prisoner, true, false);
+				target.GetExtendedPlayer().ResetAndSavePosition(getPositionT.position, getPositionT.rotation, jail.GetPlaceIndex);
+				target.svPlayer.SvClearCrimes();
+				target.RemoveItemsJail();
+				target.StartCoroutine(target.svPlayer.JailTimer(timeInSeconds));
+				target.svPlayer.Send(SvSendType.Self, Channel.Reliable, ClPacket.ShowTimer, timeInSeconds);
+				player.TS("player_jail", player.username.CleanerMessage(), player.username.CleanerMessage(), timeInSeconds);
+				player.TS("target_jail", player.username.CleanerMessage(), player.username.CleanerMessage(), timeInSeconds);
```

Comment:
I'm pretty sure this shouldn't be `player.username` 4 times.

Reply:
@ggggg, any clearance about this one? Also formatting still needs to be resolved now that you added the if == null statement.

Reply:
should be fixed now

Reply:
Sorry for late response